docs/index.md

```diff
@@ -42,3 +42,9 @@ dependencies {
 
 Artifacts of SDA Spring Boot Commons are available [at Maven Central](https://search.maven.org/search?q=g:org.sdase.commons.spring.boot)
 since release [0.11.2](https://github.com/SDA-SE/sda-spring-boot-commons/releases/tag/0.11.2).
+
+### Static directory
+Since Spring Boot runs in an embedded Tomcat server, it needs a tmp directory to support the container run in a readonly file system.
+Therefore, your application need to set a folder called `static` in the root directory of your container.
+In order to do so, create a folder `static` on `src/main/jib`.
+When the container is generated with `gradlew jibDockerBuild`, this folder will be copied to the container.
```

Comment:
I like the explanation!

But don't service developers still have to create `/tmp/tomcat` in addition to `/static` in their container?